devolutions-gateway/src/api/jrec.rs

```diff
@@ -20,14 +20,22 @@ use crate::token::{JrecTokenClaims, RecordingFileType, RecordingOperation};
 use crate::DgwState;
 
 pub fn make_router<S>(state: DgwState) -> Router<S> {
-    Router::new()
+    let router = Router::new()
         .route("/push/:id", get(jrec_push))
         .route("/delete/:id", delete(jrec_delete))
         .route("/list", get(list_recordings))
         .route("/pull/:id/:filename", get(pull_recording_file))
         .route("/play", get(get_player))
-        .route("/play/*path", get(get_player))
-        .with_state(state)
+        .route("/play/*path", get(get_player));
+
+    if state.conf_handle.get_conf().debug.enable_unstable {
+        router
+            .route("/list-active", get(list_active_recordings))
```

Comment:
suggestion: I like François' idea. Let's modify `/list` to take a query param such as "?active=true/false".

Reply:
Updated